src/restconfig/src/main/java/org/geoserver/rest/AtomLink.java

```diff
@@ -0,0 +1,66 @@
+/* (c) 2016 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.rest;
+
+import com.thoughtworks.xstream.XStream;
+/**
+ * 
+ * Can be used to encode AtomLink with X-Stream.
+ * Not using annotations because x-stream configuration is different for XML and JSON (the latter not using namespaces), 
+ * instead providing static methods for this purpose.
+ * 
+ * @author Niels Charlier
+ *
+ */
+public class AtomLink {
+
+    @SuppressWarnings("unused")
+    private final String NAMESPACE = "http://www.w3.org/2005/Atom";
+    
+    private String href, rel, type;
```

Comment:
@NielsCharlier the code conventions prefer one declaration per line (section 6.1).